app/models/agents/animated-agent.js

```diff
@@ -0,0 +1,28 @@
+// TODO: This file was created by bulk-decaffeinate.
+// Sanity-check the conversion and remove this comment.
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS206: Consider reworking classes to avoid initClass
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
```

Comment:
I don't think these suggestions apply any more and can therefore be removed. In general, where automated suggestions have already been addressed it would be good to remove them.

Reply:
:+1: